src/spyglass/position/v1/position_dlc_pose_estimation.py

```diff
@@ -271,24 +346,22 @@ def add_timestamps(df: pd.DataFrame, raw_pos_df: pd.DataFrame) -> pd.DataFrame:
     ----------
     df : pd.DataFrame
         pose estimation dataframe to add timestamps
-    raw_pos_df : pd.DataFrame
-        dataframe fetched from RawPosition that
-        contains timestamps and video frame inds
+    pos_time : np.ndarray
+        numpy array containing timestamps from the raw position object
+    video_time: np.ndarray
+        numpy array containing timestamps from the video file
 
     Returns
     -------
     pd.DataFrame
         original df with timestamps and video_frame_ind as new columns
     """
-
-    raw_pos_df = raw_pos_df.drop(
-        columns=[
-            column for column in raw_pos_df.columns if column not in ["video_frame_ind"]
-        ]
+    first_video_frame = np.where(video_time == pos_time[0])[0][0]
```

Comment:
Might want to use np.searchsorted here instead of directly comparing them because comparing floating point numbers can be tricky.